webrtc.html

```diff
@@ -83,6 +83,7 @@
     <dfn>MediaStreamConstraints</dfn> are defined in [[!GETUSERMEDIA]].</p>
     <p>The term <dfn>Blob</dfn> is defined in [[!FILEAPI]].</p>
     <p>The term <dfn>media description</dfn> is defined in [[!RFC4566]].</p>
+    <p>The term <dfn>generation</dfn> is defined in [[TRICKLE-ICE]] Section 2.</p>
```

Comment:
@stefhak Do you know why the reference to Trickle-ICE is non-normative? Isn't this mandatory-to-implement??